.github/workflows/ci.yml

```diff
@@ -17,12 +17,22 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+    - name: Install poetry
+      run: pipx install poetry
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.9'
+        cache: poetry
+    - name: Install dependencies
+      run: poetry install
     - name: Docs
       run: |
+        poetry run meowlflow openapi --schema-path=examples/document_splitter_schema.py > examples/document_splitter_schema.json
         make README.md -B
-        DIFF="$(git status --porcelain 2>/dev/null )"
-        if [ "$(echo -n "$DIFF" | wc -l)" -ne 0 ]; then
```

Comment:
`if [ "$(echo -n "$DIFF" | wc -l)" -ne 0 ]` seems to have returned false if the `$DIFF` was 0 or 1 lines long, meaning a change to a single file wouldn't prompt an error. More simply checking if the variable is empty using `if [ -z "$DIFF" ]` is more effective and correctly detects a single file change.

Reply:
👌